src/box/lua/load_cfg.lua

```diff
@@ -749,13 +749,13 @@ setmetatable(box, {
 -- `true` when box is configured.
 --
 -- Use locked() wrapper to obtain reliable results.
-local box_is_configured = false
+box.is_configured = false
```

Comment:
I would highlight the comment:

> Use locked() wrapper to obtain reliable results.

Why it gives a reliable result for a user if it doesn't for our code?

Reply:
Ok, I rephrase to use 'configured', it is more precise when we have multiple fibers.

Reply:
AFAIU comment for using locked() is misplaced, it should be placed before 'local function load_cfg(cfg)' which is just a few lines below.

Reply:
AFAIR, no, it was mine comment and I have added it intentionally. However now I'm looking into the code (859df3a71b38c8d12047de06ed50e8c040fe2985) and don't see why not to access it without locks. Maybe it was overcaution.

Please, drop the comment. I'll look around a bit more and if I'll find what is not correct with the direct access, I'll let you know.